From b10e8500d4f71cd2a2c09b00971e58fb025aefe0 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Mon, 27 Aug 2012 17:51:40 +0300
Subject: [PATCH 301/311] conf: rename c2_conf_obj's magic fields

co_{gen,con}_magic are in keeping with co_{reg,dir}_link.
---
 conf/obj.h | 8 ++++----
 conf/reg.c | 2 +-
 doc/magix  | 2 +-
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/conf/obj.h b/conf/obj.h
index 7533068..2b603ed 100644
--- a/conf/obj.h
+++ b/conf/obj.h
@@ -220,21 +220,21 @@ struct c2_conf_obj {
 	 * Generic magic.
 	 *
 	 * All configuration objects have the same value of
-	 * ->co_magic_gen.
+	 * ->co_gen_magic.
 	 *
 	 * This magic value is used for list operations.
 	 */
-	uint64_t                      co_magic_gen;
+	uint64_t                      co_gen_magic;
 	/**
 	 * Concrete magic.
 	 *
 	 * Different concrete object types have different values of
-	 * ->co_magic_con.
+	 * ->co_con_magic.
 	 *
 	 * This magic value is used for generic-to-concrete casting
 	 * (see C2_CONF_CAST()).
 	 */
-	uint64_t                      co_magic_con;
+	uint64_t                      co_con_magic;
 };
 
 /**
diff --git a/conf/reg.c b/conf/reg.c
index 51e297c..db7e9f8 100644
--- a/conf/reg.c
+++ b/conf/reg.c
@@ -42,7 +42,7 @@
 enum { REG_MAGIC = 0x33fab1edfe0da177 };
 
 C2_TL_DESCR_DEFINE(c2_conf_reg, "registered c2_conf_obj-s", ,
-		   struct c2_conf_obj, co_reg_link, co_magic_gen,
+		   struct c2_conf_obj, co_reg_link, co_gen_magic,
 		   C2_CONF_OBJ_MAGIC, REG_MAGIC);
 C2_TL_DEFINE(c2_conf_reg, , struct c2_conf_obj);
 
diff --git a/doc/magix b/doc/magix
index 4edfba7..30a7cea 100755
--- a/doc/magix
+++ b/doc/magix
@@ -39,4 +39,4 @@ c2_long_lock         l_magix                 0b055b1e55edba55 boss blessed bass
 c2_fom_thread        lt_magix                fa151f1ab1ec0b01 falsifiable COBOL
 thr_tl               td_head_magic           dec1a551f1edcade declassified cade
 c2_conf_reg          r_magic                 33fab1edfe0da177 fabled feodal
-c2_conf_obj          co_magic_gen            335e1f1e55ce1177 selfless cell
+c2_conf_obj          co_gen_magic            335e1f1e55ce1177 selfless cell
-- 
1.8.3.2

